### PR TITLE
add rbe-debian9 and rbe-ubuntu16_04 container

### PR DESCRIPTION
**Pull request by erain**
* rename debian8-clang-fully-loaded to its official name: [rbe-debian8](https://pantheon.corp.google.com/launcher/details/google/rbe-debian8)
* rbe-debian9 with almost identical packages with rbe-debian8, except for `libc++6-dev` and `ca-certificates-java` 
* rbe-ubuntu16_04 with almost identical packages with rbe-debian8, except for `ca-certificates-java` 

Tested:
* container/build.sh -p my-project -c rbe-debian8 -d debian8 -t latest -b my-debs -a
* container/build.sh -p my-project -c rbe-debian9 -d debian9 -t latest -b my-debs -a
* container/build.sh -p my-project -c rbe-ubuntu16_04 -d ubuntu16_04 -t latest -b my-debs -a
* container/build.sh -d debian8 -l
* container/build.sh -d debian9 -l
* container/build.sh -d ubuntu16_04 -l
* tested produced rbe-debian9 and rbe-ubuntu16_04 image for building gRPC and Bazel

### CODE REVIEW

File: container/build.sh [hunk 1]
```diff
@@ -23,9 +23,10 @@ Usage: build.sh [options]
 Builds the fully-loaded container, with Google Cloud Container Builder or locally.
 
 Required parameters (when build with Google Cloud Container Builder):
+    -d|--distro             Distro of the base image, debian 8, debian 9
```
Comment (nlopezgi) [inline container/build.sh]: add full list of supported base images here with the same names as expected below (debian8, debian9, ubuntu16_04)

File: container/experimental/rbe-ubuntu16_04/BUILD [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        "/usr/bin/python": "/usr/bin/python2.7",
+        "/opt/python3.6/bin/python3": "/opt/python3.6/bin/python3.6",
+        "/opt/python3.6/bin/pip3": "/opt/python3.6/bin/pip3.6",
+    },
+    tars = ["//third_party/python:tar"],
+)
+
+container_test(
+    name = "fl-toolchain-test",
+    configs = ["//container/test:rbe-ubuntu16_04.yaml"],
```
Comment (nlopezgi) [inline container/experimental/rbe-ubuntu16_04/BUILD]: Lets try to see if we can pass several files here and split off everything that is common from the verifications below to a single common file.

File: container/experimental/rbe-debian9/BUILD [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    symlinks = {
+        "/usr/bin/python": "/usr/bin/python2.7",
+        "/opt/python3.6/bin/python3": "/opt/python3.6/bin/python3.6",
+        "/opt/python3.6/bin/pip3": "/opt/python3.6/bin/pip3.6",
+    },
+    tars = ["//third_party/python:tar"],
+)
+
+container_test(
+    name = "fl-toolchain-test",
```
Comment (nlopezgi) [inline container/experimental/rbe-debian9/BUILD]: call this just "toolchain-test"

File: container/experimental/rbe-ubuntu16_04/BUILD [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+load(
+    "//container/rules:docker_toolchains.bzl",
+    "language_tool_layer",
+    "toolchain_container",
+    "cleanup_commands",
+)
+load("@io_bazel_rules_docker//contrib:test.bzl", "container_test")
+
+toolchain_container(
+    name = "fl-toolchain",
```
Comment (nlopezgi) [inline container/experimental/rbe-ubuntu16_04/BUILD]: just "toolchain"? (or rbe-toolchain or something like that?)

**Comment by erain** [comment]
I have replaced "fl-toolchain" to "toolchain" everywhere and tested the changes.

**Comment by erain** [comment]
After (many) retries the tests finally passed.  We should bring it up to bazel testing infra team, but not the priority ATM. 

I will merge this change and started testing the new images.

**Comment by vladmos** [comment]
As mentioned above, this change breaks the tests on the CI: https://buildkite.com/bazel/bazel-toolchains/builds/120
Even if the problem is with the CI itself, the failing test should be fixed somehow or switched off, otherwise it'll be easy to miss other failures in the future.

**Comment by erain** [comment]
@vladmos I have been tested the failing tests and they all passed locally
- `bazel test //container/experimental/rbe-debian9:toolchain-test`
- `bazel test //container/experimental/rbe-ubuntu16_04`

I am sending a PR to disable these two tests temporarily. However, on the other hand, is there a way for me to debug the Bazel CI system to see what might be wrong?

**Comment by vladmos** [comment]
Here's an example of the test logs, looks like the OS version string doesn't match the expected string: https://storage.googleapis.com/bazel-buildkite-artifacts/41fcd03a-3a4c-4ddd-937a-95ddf5a0df3f/container/experimental/rbe-debian9/toolchain-test/attempt_1.log

**Comment by erain** [comment]
Hi Vladimir, 

If you take a closer look, you'll notice things in the tests output were messed up. 

The test itself complain the following target failed: `//container/experimental/rbe-debian9:toolchain-test`, with wrong OS version. However, if you clone the repo and run `bazel test //container/experimental/rbe-debian9:toolchain-test`, it would just pass. 

So it looks to me that the stdout of another test, `//container/experimental/rbe-ubuntu16_04:toolchain-test`, affected this test and causeed one OR both tests failed.

**Pull request merged**